pkg/endpoint/policy.go

```diff
@@ -235,8 +298,14 @@ func (e *Endpoint) resolveL4Policy(owner Owner, repo *policy.Repository, c *poli
 }
 
 // Must be called with global endpoint.Mutex held
-func (e *Endpoint) regenerateConsumable(owner Owner, labelsMap *LabelsMap, repo *policy.Repository, c *policy.Consumable) bool {
-	changed := false
+func (e *Endpoint) regenerateConsumable(owner Owner, labelsMap *LabelsMap,
+	repo *policy.Repository, c *policy.Consumable) (bool, map[policy.NIPortProto]bool) {
+
+	var (
+		changed     = false
+		l4Rm, l4Add map[policy.NIPortProto]bool
```

Comment:
Define a type for this instead of hardcoding this map everywhere

Reply:
Done